bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/ChannelState.java

```diff
@@ -20,7 +20,9 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
See #7722 
Can StringUtils.isNotBlank be replaced by  !isBlank?

Reply:
Great idea! done.

Reply:
> I am not involved at all in the MQTT code but can"t the usual OH transformation stuff be rather used?

Neither am I. I'm just happen to need some features from mqtt. From what I can see, it is using OH transformation under the hood:

https://github.com/openhab/openhab-addons/blob/882e52647da3dfddbf35e896368c29cadc8c8a92/bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/ChannelStateTransformation.java#L20-L33

Reply:
@lolodomo, @jimtng - can this comment be considered resolved?

Reply:
Not yet @jlaur

Reply:
My question is in fact: is all the transformation addons provided by OH that can be applied by the user on any item state not sufficient?

Reply:
If you are sure this is the good approach, let say it is ok for me.

Reply:
Do you have something different in mind?